tests/unit/mocks/mockObjects.ts

```diff
@@ -12,12 +20,8 @@ export const MOCK_APP_CONFIG = {
 };
 
 export const MOCK_ADMIN1_GEOJSON = {
-    MWI: {
-        features: [{ properties: { shapeID: "123", shapeName: "Test123", shapeGroup: "MWI" } }]
-    },
-    TZA: {
-        features: [{ properties: { shapeID: "789", shapeName: "Test789", shapeGroup: "TZA" } }]
-    }
+    MWI: [{ properties: { shapeID: "123", shapeName: "Test123", shapeGroup: "MWI" } }],
+    TZA: [{ properties: { shapeID: "789", shapeName: "Test789", shapeGroup: "TZA" } }]
```

Comment:
(a question that came up while trying to rebase)
Why are you removing the 'features' property for this and not the MOCK_ADMIN2_GEOJSON? (One of my tests relies on it)

Reply:
To cut down on startup time we're now reading the admin1 geojson and indicators in from single files rather than files per-country. Since we needed to reformat the data anyway to do that, we've just pulled in the feature lists rather than the whole country level metadata that we don't need.